sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala

```diff
@@ -442,3 +627,45 @@ private[ui] class SqlStatsPagedTable(
     }
 
   }
+
+  private[ui] class SessionStatsTableDataSource(
+    info: Seq[SessionInfo],
+    pageSize: Int,
+    sortColumn: String,
+    desc: Boolean) extends PagedDataSource[SessionInfo](pageSize) {
+
+    // Sorting SessionInfo data
+    private val data = info.sorted(ordering(sortColumn, desc))
+
+    private var _slicedStartTime: Set[Long] = null
+
+    override def dataSize: Int = data.size
+
+    override def sliceData(from: Int, to: Int): Seq[SessionInfo] = {
+      val r = data.slice(from, to)
+      r.map(x => x)
```

Comment:
Nit: what does this do?

Reply:
> Nit: what does this do?

For paging, get selected pages data.

Reply:
No, I mean, `.map(x => x)` is a no-op right? what is this line for?

Reply:
> No, I mean, `.map(x => x)` is a no-op right? what is this line for?

Same implement with https://github.com/apache/spark/pull/26215

Since same function... so don't check this method. 
Remove both of these.

Reply:
Yes, we can remove the line.